src/app/api/delegate/proposal/route.ts

```diff
@@ -109,24 +112,32 @@ export async function POST(req: Request) {
         cid: cid,
```

Comment:
this signature is not being used now. We can remove it. Also, the cid is no longer needed in the body. Make sure to remove these redundant variables both from api and from where they are being sent